test/test_xhighfive.cpp

```diff
@@ -0,0 +1,73 @@
+/***************************************************************************
```

Comment:
I think it would be cool to have a third-party created file and load that with xhighfive in the tests as a sanity check. But we can also merge first.

Reply:
Will do.